src/main/java/org/pac4j/jax/rs/pac4j/JaxRsApplicationLogoutLogic.java

```diff
@@ -0,0 +1,16 @@
+package org.pac4j.jax.rs.pac4j;
+
+import org.pac4j.core.engine.DefaultApplicationLogoutLogic;
+
+/**
+ * @author Victor Noel - Linagora
+ * @since 1.1.1
+ * 
+ */
+public class JaxRsApplicationLogoutLogic<C extends JaxRsContext> extends DefaultApplicationLogoutLogic<Object, C> {
```

Comment:
OK. We're in line. Let's wait for @millross 's feedback which originally requested the custom profile manager for Vert.x.